src/Classes/Options/Interactivity/Modes/Connect.ts

```diff
@@ -0,0 +1,32 @@
+import {IConnect} from "../../../../Interfaces/Options/Interactivity/Modes/IConnect";
+import {ConnectLineLinked} from "./ConnectLineLinked";
+import {IConnectLineLinked} from "../../../../Interfaces/Options/Interactivity/Modes/IConnectLineLinked";
+
+export class Connect implements IConnect {
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new lineLinked
+     */
+    public get line_linked(): IConnectLineLinked {
+        return this.lineLinked;
+    }
+
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new lineLinked
+     * @param value
+     */
+    public set line_linked(value) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'value' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)